code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -771,51 +754,29 @@
 	overdose_threshold = REAGENTS_OVERDOSE
 	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL * 1.2 //make this a little more forgiving in light of the lethality
 	scannable = TRUE
+	taste_description = "bitterness"
 
-/datum/reagent/xeno_neurotoxin/on_mob_life(mob/living/M)
-	. = ..()
-	if(!.)
-		return
+
+/datum/reagent/xeno_neurotoxin/on_mob_life(mob/living/L, alien)
 	var/halloss_damage = volume * 2 * REM
-	M.apply_damage(halloss_damage, HALLOSS) //1st level neurotoxin effects: halloss/pain
+	L.apply_damage(halloss_damage, HALLOSS) //1st level neurotoxin effects: halloss/pain
 	if(volume > 5) //2nd level neurotoxin effects: screen shake, drug overlay, stuttering, minor toxin damage
-		M.druggy += 1.1
-		M.stuttering += 1.1
+		L.adjust_drugginess(1.1)
+		L.stuttering = max(L.stuttering, 1)
 	if(volume > 15) //3rd level neurotoxin effects: eye blur
-		M.eye_blurry += 1.1
+		L.blur_eyes(5)
 	if(volume > 20) //4th level neurotoxin effects: blindness, deafness
-		M.ear_deaf += 1.1
-		M.eye_blind += 1.1
+		L.adjustEarDamage(0, 1)
+		L.blind_eyes(5)
 	if(volume > 25) //5th level neurotoxin effects: paralysis
-		M.stunned += 1.1
-		M.KnockDown(1.1)
-
-
-/datum/reagent/xeno_neurotoxin/overdose_process(mob/living/M)
-	M.adjustOxyLoss(5) //Overdose starts applying more oxy damage
-	M.Jitter(4) //Lets Xenos know they're ODing and should probably stop.
-
-
-/datum/reagent/xeno_neurotoxin/overdose_crit_process(mob/living/M)
-	M.Losebreath(2) //Can't breathe; for punishing the bullies
-
-
-/datum/reagent/xeno_growthtoxin
-	name = "Larval Accelerant"
-	id = "xeno_growthtoxin"
-	description = "A metabolic accelerant that dramatically increases the rate of larval growth in a host."
-	reagent_state = LIQUID
-	color = "#CF3600" // rgb: 207, 54, 0
-	custom_metabolism = GROWTH_TOXIN_METARATE // 0.2, slow meta rate.
-	overdose_threshold = REAGENTS_OVERDOSE
-	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL
-	scannable = TRUE
+		L.KnockDown(1)
+	return ..()
 
 
-/datum/reagent/xeno_growthtoxin/overdose_process(mob/living/M)
-	M.adjustOxyLoss(3)
-	M.Jitter(4) //Lets Xenos know they're ODing and should probably stop.
+/datum/reagent/xeno_neurotoxin/overdose_process(mob/living/L, alien)
```

Comment:
Should also move this into the toxin.dm

Reply:
Ok, done so, that will alterate balance a little around livers and dylovene, but adjustment can be made later.

Reply:
You don't have to change the typepath, just organize it better